hack/release.sh

```diff
@@ -25,7 +25,7 @@ PIPELINE_ARTIFACTS="pkg/pipelines/resources/tekton/task/func-buildpacks/0.1/func
 
 function build_release() {
   echo "🚧 🐧 Building cross platform binaries: Linux 🐧 (amd64 / arm64 / ppc64le / s390x), MacOS 🍏, and Windows 🎠"
-  ETAG=${TAG} make cross-platform
+  FUNC_REPO_BRANCH_REF="$(git branch --show-current)" ETAG=${TAG} make cross-platform
```

Comment:
Just a question: can I use some envvar set by `../vendor/knative.dev/hack/release.sh` instead of invoking `git`?

Reply:
How branch name should look like: `release-1.11`.

Reply:
After discussion it appears `git branch --show-current` is a good solution.